packages/opentelemetry-basic-tracer/src/Span.ts

```diff
@@ -123,6 +124,22 @@ export class Span implements types.Span {
     return true;
   }
 
+  toReadableSpan(): ReadableSpan {
+    return {
+      name: this._name,
+      traceId: this._spanContext.traceId,
+      spanId: this._spanContext.spanId,
+      parentSpanId: this._parentId,
+      kind: this._kind,
+      startTime: this._startTime,
+      endTime: this._endTime,
+      status: this._status,
+      attributes: this._attributes,
+      links: this._links,
+      events: this._events,
+    };
+  }
+
   toString() {
     const json = JSON.stringify({
```

Comment:
Same comment as in other PR: can this use `toReadableSpan` above?

Reply:
sure, done.